### ISSUE TITLE
Overfitting/training instability issues

### ISSUE THREAD

**Issue by adithyamurali** [Pointcept/PointTransformerV3#139]
Hi @Gofinge thanks once again for releasing this amazing backbone.

I wanted to get your feedback on a weird issue I have been having. This only happens on some specific datasets.

I'm using ptv3 as a point cloud encoding backbone - you can just think of the loss being a L2 regression loss. On every dataset where I do not have this issue, I can get Ptv3 working better than Pointnet++. However, on some datasets, I noticed that the network overfits or has some significant training instability.

Here's an example. Yellow is PointTransformerV3 while blue is PointNet++. The top and bottom are the train and test curves effectively.

![Image](https://github.com/user-attachments/assets/af3a292e-33ec-4059-9e62-d0851543ca7c)
![Image](https://github.com/user-attachments/assets/78f235ee-b3ff-48a6-964b-4e717f7e89c5)

I'm just using the PointTransformerV3 implementation from here: https://github.com/Pointcept/PointTransformerV3/blob/main/model.py#L786 below:
```
            point_cloud_encoder = PointTransformerV3(
                in_channels=3,
                enable_flash=False,
                cls_mode=True,
            )
```

The point clouds are all within a range of [-1 to 1]

Wondering if you have any tips to further tune this.

**Comment by Gofinge** [comment]
This is an interesting phenomunion. I am not certain about what made this overfitting. Yet for some dataset with extremely small scale of data, maybe reduce the paramater size would be good. Also, you can try our self-supervised PTv3 now: https://github.com/facebookresearch/sonata

**Comment by adithyamurali** [comment]
Thanks @Gofinge for the reply. I'll try to reduce the parameter size/capacity of the network.

Also congrats on the Sonata release! I'm still trying to train a backbone from scratch on my own dataset, i guess sonata is not as relevant there?

thanks

**Comment by Gofinge** [comment]
If possible, initialize your model with Sonata. It will significant solve overfitting problems with limited data.
